Lib/_strptime.py

```diff
@@ -522,6 +523,10 @@ def _strptime(data_string, format="%a %b %d %H:%M:%S %Y"):
             # Pad to always return microseconds.
             s += "0" * (6 - len(s))
             fraction = int(s)
+        elif group_key == "F":
```

Comment:
```suggestion
        elif group_key == 'F':
```
I'd keep current style as in the rest of branches.

Reply:
Ill get back to these if the issue is not resolved without a pr.